src/BaGetter.Core/Configuration/AuthenticationOptions.cs

```diff
@@ -0,0 +1,8 @@
+namespace BaGetter.Core;
+
+public class AuthenticationOptions
+{
+    public string Username { get; set; }
+
+    public string Password { get; set; }
```

Comment:
Well. Thats given you would like to implement login screen, user management screen etc. This was intended to serve purpose to of deploying the Bagetter with (docker compose, or appsettings on IIS) and with preset Username/Password (like with api key), to prevent unauthorized download.

That doesnt mean that could not be extended in the future to work for both (this as system/default users and additional user profiles created later on and stored in DB). But that was not t

Reply:
Hm that makes sense, and this is way easier than implementing whole oauth workflows. Any idea if we can do this without the passwords? Can't give the admin your actual password.   
Turns out I was stuck on semantics. Could just call it a personal API key and I'm happy. Unfortunately though nuget calls it password so naturally, writing it down or sharing it feels wrong. 

Make sure to add it to the documentation, then let's see if we can find some more people with opinions on this.

Reply:
I was thinking about not calling it directly username and password, but the Nuget CLI is the way it is and that would be more confusing than not calling it that way...

For example with telerik private feeds you need to use your actual username and password when you set up the feed. The admin will just "make up" the username and password. 

Then I think next logical step is to open config/way, to let people implement their own auth using maybe HTTP call, or smthing. But that will be much harder than this

Reply:
I have updated the documentation on private feeds

Reply:
Also for `Any idea if we can do this without the passwords? Can't give the admin your actual password.`
The username would lean to be "quessable", I think its much better to be provided with memorizeable username, like domain username for PC and then randomly generated password, same with the api key.

But calling it username/apiKey and then setup username/password in nuget CLI seems counter intuitive